.circleci/config.yml

```diff
@@ -397,16 +400,14 @@ jobs:
     - run:
         name: Build all Kuma binaries (such as, kumactl, kuma-cp, kuma-dp, kuma-prometheus-sd)
         command: make build
-    - setup_remote_docker:
-        version: *docker_version
```

Comment:
Why removing this? I believe it helps with some layer caching.

Reply:
We are not using docker-based executor so it doesn't work.